prefetch.bs

```diff
@@ -378,8 +378,23 @@ The user agent may [=prefetch record/cancel and discard=] records from the [=Doc
 
        <p class="note">An implementation might wish to do this if it believes that the prefetch will be consumed more than once. For example, if in the future the response is consumed by a prerender, that [=prerendering traversable=] might be [=destroy a top-level traversable|destroyed=] through various means. Normally that would mean the response is discarded, but if the implementation performs this step, then the prefetched response will still be available to serve a future navigation. [[PRERENDERING-REVAMPED]]
     1. If the user agent did not perform the previous optional step, then it must [=list/remove=] |record| from |navigable|'s [=navigable/active document=]'s [=Document/prefetch records=].
-
-
+    1. <span id="prefetch-activation-client-creation"></span>Let |reservedEnvironment| be a new [=environment=] with:
+        : [=environment/id=]
+        :: a unique opaque string
+        : [=environment/creation URL=]
+        :: |request|'s [=request/reserved client=]'s [=environment/creation URL=]
+        : [=environment/top-level creation URL=]
+        :: |request|'s [=request/reserved client=]'s [=environment/top-level creation URL=]
+        : [=environment/top-level origin=]
+        :: |request|'s [=request/reserved client=]'s [=environment/top-level origin=]
+        : [=environment/target browsing context=]
+        :: |request|'s [=request/reserved client=]'s [=environment/target browsing context=]
+        : [=environment/active service worker=]
+        :: |request|'s [=request/reserved client=]'s [=environment/active service worker=]
```

Comment:
I think `request's active service worker` is never set, because the `request` doesn't go through actual fetching.
Probably we want to clone the prefetch record's last leg's request's active service worker instead?

Reply:
I guess only the active service worker changes, so probably that's simpler...